### DISCUSSION TITLE
Is there a way to use an image with transparency in the Maps sample?

### DISCUSSION THREAD

**Discussion by badsonstudios** [VladislavAntonyuk/MauiSamples#172] [Q&A] [answered]
I've tried using an svg image, but it seems to fail and only the default pin is shown.
I also tried a png with transparency, but the transparent sections are still a solid color.

**Comment by VladislavAntonyuk** [comment, marked as answer]
It should work out of the box. Could you please share the image or sample app? also what platform do you use?

**Reply by badsonstudios** [reply]
I'm using the maps sample with no changes other than the maps API Key.  The pin.svg file is in the Resources\EmbeddedImages folder.

var customPinFromResource = new CustomPin()
{
	Label = "From Resource",
	Location = new Location(12, 12),
	Address = "Address3",
	ImageSource = ImageSource.FromResource("MauiMaps.Resources.EmbeddedImages.pin.svg"),
	Map = MyMap
};

On iOS Simulator (iOS 14), I don't see any pins at all.  And I left in the ImageSource.FromUri pin.
ImageSource = ImageSource.FromUri(new Uri("https://picsum.photos/50")),

On Android I see 2 pins, but the svg pin is the default Android map pin (screenshot attached).
<img width="220" alt="Android_pins" src="https://user-images.githubusercontent.com/9747189/207449727-d1b8d571-9fce-4d26-816a-419fc09148ae.png">

SVG image:

![pin](https://user-images.githubusercontent.com/9747189/207446858-741192e8-40f3-4d1e-8589-f8834a73d9a8.svg)

**Reply by VladislavAntonyuk** [reply]
![image](https://user-images.githubusercontent.com/33021114/207458785-fb0625eb-7da2-4863-b5a7-7303d5b0573d.png)
you need to convert image to png

**Comment by badsonstudios** [comment]
Yup, when I saved the svg as a png, it was not including transparency.  Thank you!
